libs/community/langchain_community/graphs/neo4j_graph.py

```diff
@@ -31,6 +31,34 @@
 """
 
 
+def value_sanitize(d: Dict[str, Any]) -> Dict[str, Any]:
```

Comment:
in the end does this just remove lists longer than LIST_LIMIT?

Reply:
Yes

Reply:
could we explain that in docstring

Reply:
Added